detekt-rules-exceptions/src/test/kotlin/io/gitlab/arturbosch/detekt/rules/exceptions/SwallowedExceptionSpec.kt

```diff
@@ -134,15 +145,58 @@ class SwallowedExceptionSpec : Spek({
             assertThat(subject.compileAndLint(code)).hasSize(1)
         }
 
-        listOf(listOf("IllegalArgumentException"), "IllegalArgumentException").forEach { ignoredExceptionValue ->
-            context("ignores given exception types config") {
+        @Nested
+        inner class `listOf(IllegalArgumentException)` {
```

Comment:
```suggestion
        inner class `when given listOf(IllegalArgumentException)` {
```

nit: This reads really odd to me. I would rephrase to be a bit more verbose.

Reply:
(similar for the other nested class)